<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestBGMSetting</class>
 <widget class="QDialog" name="TestBGMSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>背景音乐</string>
  </property>
  <widget class="QSlider" name="setMusicPitchSlider">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>161</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="setVoiceReverbTypeComboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>Type_0 关闭混响</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Type_1 KTV</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Type_2 小房间</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Type_3 大会堂</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Type_4 低沉</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Type_5 洪亮</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Type_6 金属声</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Type_7 (磁性)</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>播放背景音乐</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="loopCountEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>386</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>人声混响效果：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>99</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>本地音量：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>播放音调：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="setMusicPlayoutVolume">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>150</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>播放变速：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="setMusicSpeedRateSlider">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="getMusicDurationInMS">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>440</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>00 ms</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="setAllMusicVolume">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>129</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>150</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>循环次数:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="playOrStopBGM">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>420</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>129</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>全局音量： </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置背景音乐</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pauseBGM">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>261</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>   重置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>远端音量： </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="setMusicPublishVolume">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>150</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>381</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>381</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置短音效</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>540</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>音效1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>540</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>循环</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>540</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>推送远端</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="soundPlay1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>540</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="soundReplay1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>540</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="soundPush1">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>540</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListView" name="listView_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>381</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>570</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>推送远端</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="soundReplay2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>570</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="soundPush2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>570</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>570</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>音效2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>570</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>循环</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="soundPlay2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>570</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>600</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>音效3</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="soundPush3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>600</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>600</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>推送远端</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>600</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>循环</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="soundReplay3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>600</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="soundPlay3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>600</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="bgmSlider">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>410</y>
     <width>181</width>
     <height>40</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <zorder>listView_3</zorder>
  <zorder>listView_2</zorder>
  <zorder>listView</zorder>
  <zorder>setMusicPitchSlider</zorder>
  <zorder>setVoiceReverbTypeComboBox</zorder>
  <zorder>label_8</zorder>
  <zorder>loopCountEdit</zorder>
  <zorder>label_7</zorder>
  <zorder>label_3</zorder>
  <zorder>label_5</zorder>
  <zorder>setMusicPlayoutVolume</zorder>
  <zorder>label_6</zorder>
  <zorder>setMusicSpeedRateSlider</zorder>
  <zorder>getMusicDurationInMS</zorder>
  <zorder>setAllMusicVolume</zorder>
  <zorder>label_9</zorder>
  <zorder>playOrStopBGM</zorder>
  <zorder>label_4</zorder>
  <zorder>label_11</zorder>
  <zorder>pauseBGM</zorder>
  <zorder>resetButton</zorder>
  <zorder>label_2</zorder>
  <zorder>setMusicPublishVolume</zorder>
  <zorder>label_10</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>soundPlay1</zorder>
  <zorder>soundReplay1</zorder>
  <zorder>soundPush1</zorder>
  <zorder>label_15</zorder>
  <zorder>soundReplay2</zorder>
  <zorder>soundPush2</zorder>
  <zorder>label_16</zorder>
  <zorder>label_17</zorder>
  <zorder>soundPlay2</zorder>
  <zorder>label_18</zorder>
  <zorder>soundPush3</zorder>
  <zorder>label_19</zorder>
  <zorder>label_20</zorder>
  <zorder>soundReplay3</zorder>
  <zorder>soundPlay3</zorder>
  <zorder>bgmSlider</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
